fsevents.js

```diff
@@ -6,6 +6,13 @@
 /* jshint node:true */
 'use strict';
 
+if (process.platform !== 'darwin') {
+  var err = new Error('Cannot find module \'fsevents\'');
```

Comment:
Most likely, dependents react to some exception being thrown, rather than parsing the actual message for some reason. They might query `err.code` for `'MODULE_NOT_FOUND'` just to be sure.

Would this be an acceptable middle ground to you?
```javascript
if (process.platform !== 'darwin')
   require('fsevents-' + process.platform)
```

This would throw a very similar error, and is technically accurate.